mitodl/bootcamp-ecommerce
# update the favicon

#### What are the relevant tickets?
#736 

#### What's this PR do?
#fixes #736 , Update the favicon to use the ones from https://bootcamps.mit.edu/ i.e. https://bootcamp.mit.edu/wp-content/uploads/2018/08/Bootcamp-Symbol-Logo-Large-150x150.png and https://bootcamp.mit.edu/wp-content/uploads/2018/08/Bootcamp-Symbol-Logo-Large.png

#### How should this be manually tested?
Need to restart your docker containers to see the updated bundle with latest favicon.

#### Screenshots (if appropriate)
<img width="259" alt="Screenshot 2020-06-23 at 18 04 13" src="https://user-images.githubusercontent.com/4043989/85407800-1ec52f00-b57d-11ea-91e6-5624645a048b.png">
<img width="1402" alt="Screenshot 2020-06-23 at 18 12 13" src="https://user-images.githubusercontent.com/4043989/85407809-21c01f80-b57d-11ea-9b9e-62255956d998.png">
